fvtest/gctest/StartupManagerTestExample.cpp

```diff
@@ -84,7 +84,12 @@ MM_StartupManagerTestExample::parseLanguageOptions(MM_GCExtensionsBase *extensio
 				} else if (0 == strcmp(attr.name(), "gcthreadCount")) {
 					/* TODO: support multi-thread GC*/
 				} else if (0 == strcmp(attr.name(), "GCPolicy")) {
+#if defined(OMR_GC_MODRON_SCAVENGER)
 					extensions->scavengerEnabled = (0 == j9_cmdla_stricmp(attr.value(), "gencon"));
+#if defined(OMR_GC_MODRON_CONCURRENT_MARK)
+					extensions->concurrentMark = extensions->scavengerEnabled;
```

Comment:
Made this change: 
```
extensions->concurrentMark = extensions->scavengerEnabled || (0 == j9_cmdla_stricmp(attr.value(), "optavgpause"));
```
 and added a gctest configuration for this (optavgpause_GC_config.xml). 

All test configurations pass, and ConcurrentGC is instantiated as concurrent collector in optavgpause and gencon configurations, but more work needs to be done to GCTest allocation helper before allocation tax is collected. As mentioned in a previous commit message I intend to get back and finish GCTest integration with ConcurrentGC in a new issue.